duckplayer/duckplayer-impl/src/main/java/com/duckduckgo/duckplayer/impl/DuckPlayerDataStore.kt

```diff
@@ -194,7 +194,7 @@ class SharedPreferencesDuckPlayerDataStore @Inject constructor(
     private val duckPlayerOpenInNewTab: Flow<Boolean>
         get() = store.data
             .map { prefs ->
-                prefs[Keys.DUCK_PLAYER_OPEN_IN_NEW_TAB] ?: false
+                prefs[Keys.DUCK_PLAYER_OPEN_IN_NEW_TAB] ?: true
```

Comment:
nit: What’s the reason for defaulting to `true` here when the toggle defaults to `false`?